Sources/Datadog/Datadog.swift

```diff
@@ -158,8 +162,6 @@ public class Datadog {
         static let DebugRUM = "DD_DEBUG_RUM"
     }
 
-    internal static var instance: Datadog?
-
     internal let consentProvider: ConsentProvider
     internal let userInfoProvider: UserInfoProvider
     internal let launchTimeProvider: LaunchTimeProviderType
```

Comment:
Why we moved `consentProvider` and `userInforProvider` to `DatadogCore`, but not yet `launchTimeProvider`?

Reply:
I've noticed that `Datadog.instance?.launchTimeProvider` was never in used, it is injected as part of the `FeaturesCommonDependencies`.